src/Sentry.Log4Net/Sentry.Log4Net.csproj

```diff
@@ -1,12 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net462</TargetFrameworks>
+    <TargetFrameworks>net6.0;netstandard2.0;net462</TargetFrameworks>
```

Comment:
Lets not add net6.0 for `log4net` if we can't get away without it

Another library mainly legacy, I expect most apps using this are on .NET Framework or not really evolving quickly

Reply:
> Lets not add net6.0 for `log4net` if we can't get away without it

If we want to turn on `IsAotCompatible` for this integration then it needs to target `net6.0`, since one of the conditions for AOT compatibility is the target framework being `net6.0` or higher.